packages/tokens/raw/color_palette.json

```diff
@@ -681,201 +681,7 @@
       },
       "700": {
         "Background": {
-          "value": "#4c9599",
```

Comment:
This huge chunk which is removed is unneccesary cruft? Otherwise, this is a no-brainer